versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/azure-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Installing Rancher Prime PAYG on Azure
+---
+
+This page covers installing the Rancher Prime PAYG offering on Microsoft's Azure Marketplace.
+
+## How to Install Rancher Prime PAYG
+
+Refer to the following steps for creating a new deployment of Rancher Prime from the Azure Marketplace page.
+
+1. Select the **Rancher Prime with 24x7 Support** offer (either **EU and UK only** or **non-EU and non-UK only**) that corresponds to the location where your account is registered.
+1. Choose the plan from the dropdown list. View the **Plans + Pricing** tab for more details about the plan.
+1. Select **Create**.
+
+### Basics
+
+On the **Basics** tab, specify the **Project details** and **Instance details**:
+
+![Basics tab](/img/install-rancher-prime-basics.png)
+
+1. Select an existing **Subscription** from the dropdown list.
+1. Select an existing **Resource group** from the dropdown list.
+
+   :::note
+
+   The **Create new** resource group feature is not supported.
+
+   ![Create new resource group not supported](/img/install-rancher-prime-basics-create-new.png)
+
+   :::
+
+1. Select an existing **AKS Cluster Name** from the dropdown list.
+1. Choose an **Extension Resource name**. It can consist of alphanumeric characters and dots; the length must be between 2 and 253 characters.
```

Comment:
```suggestion
1. Choose an **Extension Resource name**. It can consist of alphanumeric characters and dots and must be between 2 and 253 characters long.
```